alerter/src/main/res/layout/alerter_alert_view.xml

```diff
@@ -1,77 +1,85 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout
-    android:id="@+id/flAlertBackground"
-    style="@style/AlertStyle"
-    xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:background="@android:color/darker_gray"
-    android:clickable="true"
-    android:foreground="?attr/selectableItemBackground"
-    tools:visibility="visible">
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+             xmlns:tools="http://schemas.android.com/tools"
+             android:layout_width="match_parent"
+             android:layout_height="match_parent"
+             android:id="@+id/flClickShield"
+             android:background="@android:color/transparent"
+             android:clickable="false">
 
-    <RelativeLayout
-        android:id="@+id/rlContainer"
-        android:layout_width="wrap_content"
-        android:layout_height="match_parent"
-        android:layout_gravity="center_vertical"
-        android:layout_marginTop="@dimen/alerter_padding_half"
-        android:background="@null"
-        android:clipChildren="false"
-        android:clipToPadding="false"
-        android:padding="@dimen/alerter_activity_horizontal_margin">
+    <FrameLayout
```

Comment:
This framelayout and the one above it have different ordering of attributes. Very minor but can be nice to have everything using a consistent order